types/node/stream/consumers.d.ts

```diff
@@ -1,22 +1,10 @@
-// Duplicates of interface in lib.dom.ts.
-// Duplicated here rather than referencing lib.dom.ts because doing so causes lib.dom.ts to be loaded for "test-all"
-// Which in turn causes tests to pass that shouldn't pass.
-//
-// This interface is not, and should not be, exported.
-interface Blob {
-    readonly size: number;
-    readonly type: string;
-    arrayBuffer(): Promise<ArrayBuffer>;
-    slice(start?: number, end?: number, contentType?: string): Blob;
-    stream(): NodeJS.ReadableStream;
-    text(): Promise<string>;
-}
 declare module 'stream/consumers' {
+    import { Blob as NodeBlob } from "node:buffer";
```

Comment:
Is the import alias necessary here?

Reply:
It isn't but in my own projects I've made a habit of aliasing anything that shadows a global type name, just to be sure there is never the possibility of confusion.  Happy to change it back if that's a style preference for the project or whatever.

Reply:
I'm not concerned. you can go ahead and ask the bot to merge.